sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -265,7 +266,7 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
     // If withdrawal epoch switched to the next one, then perform some database clean-up:
     // 1) remove outdated topQualityCertificate retrieved 3 epochs before and referenced to the 4 epochs before.
     //    Note: we should keep last 2 epoch certificates, so in case SC has ceased we have an access to the last active cert.
-    // 1) remove outdated BlockFeeInfo records
+    // 1) remove outdated AccountBlockFeeInfo records
```

Comment:
Now i have a doubt if is it safe to remove outdated `AccountBlockFeeInfo`.
Can it have an impact in case we need to debug_block  (eth rpc call) with fee payments in the past?

Reply:
Let's discuss it

Reply:
let's add todo

Reply:
Done